docs/make.jl

```diff
@@ -54,8 +57,9 @@ example_pages = [
     "Ocean wind mixing and convection" => "generated/ocean_wind_mixing_and_convection.md",
     "Langmuir turbulence"              => "generated/langmuir_turbulence.md",
     "Eady turbulence"                  => "generated/eady_turbulence.md",
-    "Kelvin-Helmholtz instability"     => "generated/kelvin_helmholtz_instability.md"
-]
+    "Kelvin-Helmholtz instability"     => "generated/kelvin_helmholtz_instability.md",
+    "Bickley jet shallow water model"     => "generated/Bickley_jet_shallow_water.md"
```

Comment:
```suggestion
    "Bickley jet shallow water model"  => "generated/Bickley_jet_shallow_water.md"
```

[to align with the rest ;)]

Reply:
Of course!  Sorry I missed that